### PR TITLE
Fix forna stylesheet

### PR DESCRIPTION
**Pull request by shammamah**
## About 

The `fornac` package includes stylesheets that apply to all SVG elements (see https://github.com/ViennaRNA/fornac/blob/9771d6d79d400e334bb306387d5fc795097ff94d/app/styles/fornac.css#L1); this package was forked to https://github.com/plotly/fornac with an additional `.forna-container` class selector prepended to each selector in the included stylesheets. 

The `FornaContainer` component in this library was accordingly updated to include a `className` property of `.forna-container` so that the styles from the original library would still apply.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -30,7 +30,7 @@
   "dependencies": {
     "circos": "git+https://github.com/matthewchan15/circosJS.git#matthewchan15-zoom-pan-svg",
     "fast-memoize": "^2.5.1",
-    "fornac": "^1.1.8",
+    "fornac": "git://github.com/plotly/fornac.git",
```
Comment (alexcjohnson) [inline package.json]: Let's put a hash here, so if and when we upgrade it's intentional :)
Same for all the git-referenced packages in fact. Even the circos branch reference isn't unique, a commit hash is better (and while I'm looking at this... much as we appreciate @matthewchan15's work, circos should probably be moved into the plotly namespace as well)

**Comment by Marc-Andre-Rivet** [comment]
Running dash-docs with the latest / modified version of DAQ and the fixed version of Bio causes problems when rendering the DAQ slider
image

image

With styles that can be traced back to `forna`
image

**Comment by Marc-Andre-Rivet** [comment]
https://github.com/ViennaRNA/fornac/blob/master/app/styles/fornac.css

File: src/lib/components/FornaContainer.react.js [hunk 2]
```diff
@@ -83,6 +83,7 @@ export default class FornaContainer extends Component {
         return (
             <div
                 id={this.props.id}
+                className="forna-container"
```
[the remark this answers is not in the document]
Comment (alexcjohnson) [reply src/lib/components/FornaContainer.react.js]: we can get rid of this line now, right?

[the remark this answers is not in the document]
Comment (shammamah) [reply src/lib/components/FornaContainer.react.js]: Ah that's right! Fixed in 8cad5f4.

[the remark this answers is not in the document]
Comment (shammamah) [reply src/lib/components/FornaContainer.react.js]: Ah that's right! Fixed in 8cad5f4.

File: .circleci/config.yml [hunk 3]
```diff
@@ -67,9 +66,10 @@ jobs:
 
         steps:
             - checkout
+            - run: echo $PYVERSION > ver.txt
 
             - restore_cache:
-                key: deps1-{{ .Branch }}-{{ checksum "tests/requirements.txt" }}
+                key: deps1-{{ .Branch }}-{{ checksum "package.json" }}-{{ checksum "ver.txt" }}
```
Comment (alexcjohnson) [inline .circleci/config.yml]:
```suggestion
                key: deps1-{{ .Branch }}-{{ checksum "tests/requirements.txt" }}-{{ checksum "ver.txt" }}
```

And `save_cache` needs updating to match

**Pull request merged**